### ISSUE TITLE
[mot rules] false negative, many rules not triggered

### ISSUE THREAD

**Issue by chris-fuchs** [angular-eslint/angular-eslint#1110]
For some reason unkown to me, most rules are not beeing considered.

Not working: button has type, img has alttext, several semicolons
working: eqeqeq, Lifecycle methods should not be empty

[I will add more and more to see if there is a pattern]

see: 
https://github.com/chris-fuchs/linterTestApp/blob/main/src/app/components/test/test.component.html
and
https://github.com/chris-fuchs/linterTestApp/blob/main/src/app/components/test/test.component.ts

JSON:
https://github.com/chris-fuchs/linterTestApp/blob/main/.eslintrc.json

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/eslint-plugin`    | `5.33.1` |
| `@typescript-eslint/parser`        | `5.33.1` |
| `ESLint`                           | `8.22.0` |
| `node`                             | `17.0.1` |
|  `@angular-eslint/builder`                             | `14.0.3-alpha.0` |
|  `@angular-eslint/eslint-plugin-template`     | `14.0.3-alpha.0` |
|  `@angular-eslint/eslint-plugin`                    | `14.0.3-alpha.0` |
|  `@angular-eslint/schematics`                      | `14.0.3-alpha.0` |
|  `@angular-eslint/template-parser`            | `14.0.3-alpha.0` |

```sh
# Please run `npx ng version` in your project and paste the full output here:

Angular CLI: 14.1.2
Node: 17.0.1 (Unsupported)
Package Manager: npm 8.1.0 
OS: linux x64

Angular: 14.1.2
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1401.2
@angular-devkit/build-angular   14.1.2
@angular-devkit/core            14.1.2
@angular-devkit/schematics      14.1.2
@schematics/angular             14.1.2
rxjs                            7.4.0
typescript                      4.7.4
```

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.
- [X] I have tried several both Angular 13 and 14 with several versions of Angular-Eslint

**Comment by JamesHenry** [comment]
Hi @chris-fuchs, I've read through this issue a couple of times and I believe I now understand your concern to be this: you would prefer it if the `recommended` configs had more rules included in them?

Just as a reminder, `recommended` is a very, very high bar - rules in them need to be universally applicable. Also please note rules like `semi` and `max-len` which you have cited above will _never_ be contained in recommended lint configs because they are to do with code formatting, and a linter is not the right tool for that. Please check out our stance on code-formatting here: [docs/FORMATTING_RULES.md](https://github.com/angular-eslint/angular-eslint/blob/main/docs/FORMATTING_RULES.md)

Please do feel free to open a discussion if you have rules you would like to be included in the recommended configs in the next major version: https://github.com/angular-eslint/angular-eslint/discussions. Please provide clear and objective reasoning as to why you feel they are recommended for everyone.

One thing we will definitely look at for the next major version is generating configs which extend from the `eslint` and `typescript-eslint` recommended configs in addition to `angular-eslint`'s own configs.

As far as I can tell from your description, everything is functioning as intended, so I'm going to close this one for now
